wdk-ddi-src/content/fltkernel/nf-fltkernel-flt_is_fs_filter_operation.md

```diff
@@ -0,0 +1,92 @@
+---
+UID: NF:fltkernel.FLT_IS_FS_FILTER_OPERATION
+tech.root: ifs
```

Comment:
ARGGGGGG! I just realized that I gave you the wrong tech.root for file systems!! Instead of "ifs", it should be "ifsk". Can you change all instances easy enough? Or would you prefer that I do it since it was my bad? So sorry!

Reply:
No worries - I'll fix it! :)